README.md

```diff
@@ -93,6 +93,34 @@ For example, on Fedora:
 
 - to add more python dependencies: add to `requirements.txt` and `requirements-workers.txt`
 
+## Database Migrations
+
+Follow the steps below for database data and/or schema migrations:
+
+- Checkout the master branch and ensure no schema changes are present in `cachito/web/models.py`
```

Comment:
Could you please use `*` instead of `-`? Markdown linter prefers them.

Reply:
Noticed in some other parts of the README as well (Not part of my changes), changing them as well.